packages/icons/manifest.json

```
@@ -370,6 +370,11 @@
       "category": "Payment method",
       "size": "24"
     },
+    {
+      "name": "postfinance",
```

Let's make this `post_finance` so it ends up being `<PostFinance />` (just like `pay_pal`) 🙂 

```suggestion
      "name": "post_finance",
```

---

Good catch, I didn't realize the brand name was camelcase haha done!

---

I just know this because it's a payment method from 🇨🇭 

brings back memories 😛